src/lib.rs

```diff
@@ -295,6 +295,17 @@ struct Inner {
     task_tx: mpsc::Sender<Task>,
 }
 
+impl std::fmt::Debug for Inner {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "LeakyBucket {{ tokens: {}, max: {}, refill_interval: {:?}, refill_amount: {} }}",
```

Comment:
I'd recommend making use of [`debug_struct`](https://doc.rust-lang.org/std/fmt/struct.Formatter.html#method.debug_struct), it should give the same result.

Reply:
TIL :)

Will fix.